### PR TITLE
The email on close code for review

### PR DESCRIPTION
**Pull request by SteveLinden**
## A reference to the issue / Description of it

See [Automation to provide info for new onboarders#3451](https://github.com/ministryofjustice/modernisation-platform/issues/3451) got full details

## How does this PR fix the problem?

Sends the email once the request is closed

## How has this been tested?

Used act to test, initially, via "act -W 'email-requester-onboarding-close.yml'"

Initial errors corrected in updated version.

## Deployment Plan / Instructions

No impact to other services

## Checklist (check `x` in `[ ]` of list items)

- [x] I have performed a self-review of my own code
- [x] All checks have passed

## Additional comments (if any)

This will be tested by creating a dummy onboarding release and seeing if it sends an email to the person listed as the owner (me). Should only take place on close

### CODE REVIEW

**Comment by SteveLinden** [comment]
Code amended to correct an error on the if statement.

File: .github/workflows/email-requester-onboarding-close.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    types:
+      - closed
+
+jobs:
+  send-message:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
```
Comment (ASTRobinson) [inline .github/workflows/email-requester-onboarding-close.yml:14]: could this be updated to v4? 
actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1

File: .github/workflows/email-requester-onboarding-close.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 14
```
Comment (ASTRobinson) [inline .github/workflows/email-requester-onboarding-close.yml:19]: bump to version 20?

File: .github/workflows/email-requester-onboarding-close.yml (hunk 1 above)
Comment (SteveLinden) [reply .github/workflows/email-requester-onboarding-close.yml]: Amended

File: .github/workflows/email-requester-onboarding-close.yml (hunk 2 above)
Comment (SteveLinden) [reply .github/workflows/email-requester-onboarding-close.yml]: Amended

**Pull request merged**